GraphConsoleAppV3/GraphRequests.cs

```diff
@@ -0,0 +1,1534 @@
+                    Password = "changeMe!",
+                    ForceChangePasswordNextLogin = true
+                };
+                newUser.PasswordProfile = PasswordProfile;
+                newUser.PasswordPolicies = "DisablePasswordExpiration, DisableStrongPassword";
+                try
+                {
+                    await newUser.UpdateAsync();
+                    Console.WriteLine("\nUser {0} password and policy was reset", newUser.DisplayName);
+                }
+                catch (Exception e)
+                {
+                    Program.WriteError("\nError Updating the user {0} {1}", e.Message,
+                        e.InnerException != null ? e.InnerException.Message : "");
+                }
+            }
+
+            #endregion
+        }
+
+        private static async Task OtherUserWriteOperations(IActiveDirectoryClient client, IUser newUser,
+            IUser signedInUser)
+        {
+            #region Other user write operations on a newly created user
+
+            // *************************************************************
+            // These operations require more privileged permissions like Directory.ReadWrite.All or Directory.AccessAsUser.All
+            // Update signed in user's manager, update group membership
+            // **************************************************************
+
+            #region Assign a manager
+
+            // Assign the newly created user a new manager (the signed in user).
+            if (newUser.ObjectId != null)
+            {
+                Console.WriteLine("\n Assign User {0}, {1} as Manager.", signedInUser.DisplayName,
+                    newUser.DisplayName);
+                newUser.Manager = signedInUser as DirectoryObject;
+                try
+                {
+                    await newUser.UpdateAsync();
+                    Console.Write("User {1} is successfully assigned {0} as their Manager.", signedInUser.DisplayName,
+                        newUser.DisplayName);
+                }
+                catch (Exception e)
+                {
+                    Program.WriteError("\nError assigning manager to user. {0} {1}", e.Message,
+                        e.InnerException != null ? e.InnerException.Message : "");
+                }
+            }
+            else
+                Console.WriteLine("\n Assigning manager failed, because new user was not created.");
+
+            #endregion
+
+            #region Add the new user to a selected group
+
+            // Search for a group and assign the newUser to the found group
+
+            //*********************************************************************
+            // Search for a group using a startsWith filter (displayName property)
+            //*********************************************************************
+            Group retrievedGroup = new Group();
+            Console.WriteLine("\nSearch for a group to add the current user to:");
```

Comment:
We're searching by group name, but for a group

---
In reply to: [80544949](https://github.com/Azure-Samples/active-directory-dotnet-graphapi-console/pull/37#discussion_r80544949) [](ancestors = 80544949)